carbonmark/components/Layout/styles.ts

```diff
@@ -80,8 +81,25 @@ export const layoutChildrenContainer = css`
   gap: 1.2rem 0rem;
   padding: 1.6rem;
   align-content: flex-start;
+
+  &:has(.feature-banner):not(.initial-banner) {
+    padding: 17rem 2.4rem;
+  }
+
+  &:has(.feature-banner.initial-banner) {
+    padding: 20rem 2.4rem;
+  }
+
+  // required for feature banner transition
+  transition: all 0.2s ease-in-out;
+
   ${breakpoints.desktop} {
     padding: 4rem 2.4rem;
+
+    &:has(.feature-banner.initial-banner),
+    &:has(.feature-banner):not(.initial-banner) {
+      padding: 16rem 2.4rem;
+    }
```

Comment:
This looks like page specific logic is bleeding in to the general Layout component and we're breaking "separation of concerns".

Not a massive deal but ideally the Layout component can accept an additional custom styling per instance and then this specific css is passed to it in the `Projects/index.tsx` component.

```tsx
<Layout customCss={styles.projectBannerPadding}/>
```

Reply:
Thanks @0xemc - great point on the separation of concerns here and something I missed. I've updated the code to move the styles out of the layout css and passed to the layout through a `customCss` prop.